lesson06/assignment/accordion.html

```diff
@@ -0,0 +1,124 @@
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="images.html">Images</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="grid.html">Grid System</a>
+                            </li>
+                            <li class="nav-item">
+                                <a class="nav-link" href="accordion.html">Accordion</a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </nav>
+        </header>
+        <section class="main_accordion">
+            <div id="accordion">
+                <div class="card">
+                    <div class="card-header">
+                        <a href="#one" data-parent="#accordion"
+                            data-toggle="collapse">
+                            <img src="img/anime-1.jpg" alt="">
+                        </a>
+                    </div>
+                    <div id="one" class="collapse">
+                        <div class="card-body">
+                            <h3 class="display-4" data-toggle="modal" data-target=".bd-example-modal-lg">Your Name</h3>
+                            <div class="modal fade bd-example-modal-lg" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel" aria-hidden="true">
+                                <div class="modal-dialog modal-lg">
+                                    <div class="modal-content">
+                                        A teenage boy and girl embark on a quest to meet each other for the<br> first time after they magically swap bodies.<br>
+                                        Release date: April 7, 2017 (USA)<br>
+                                        Director: Makoto Shinkai<br>
+                                        Japanese: 君の名は。<br>
+                                        Featured song: Zenzenzense<br>
+                                        Did you know: "Your Name" is the second-highest-grossing anime film of <br>all time worldwide ($361,024,012). wikipedia.org<br>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-header">
+                        <a href="#two"
+                            data-parent="#accordion" data-toggle="collapse">
+                            <img src="img/anime-2.jpg" alt="">
+                        </a>
+                    </div>
+                    <div id="two" class="collapse">
+                        <div class="card-body">
+                            <h3 class="display-4" data-toggle="modal" data-target=".bd-example-modal-lg">Weathering with You</h3>
```

Comment:
For your 2nd and 3rd modals to work, you'll need to give them a different class `bd-example-modal-lg2`, `bd-example-modal-lg3` or something. Then update the `data-target` to match the new classes -- right now only the first modal opens for all 3 buttons.